tomviz/AddPythonTransformReaction.cxx

```diff
@@ -228,6 +339,46 @@ OperatorPython* AddPythonTransformReaction::addExpression(DataSource* source)
           source->addOperator(op);
       }
   }
+    
+  else if (scriptLabel == "Delete Slices")
+  {
+      vtkTrivialProducer *t = vtkTrivialProducer::SafeDownCast(
+                                                               source->producer()->GetClientSideObject());
```

Comment:
Not sure of the point of the newline here - doesn't reduce line length below 80 or make it more readable. Typo?